### PR TITLE
HPCC-13236 Make query list checking status on all roxie nodes optional

### PR DESCRIPTION
**Pull request by afishbeck**
Signed-off-by: Anthony Fishbeck anthony.fishbeck@lexisnexis.com

### CODE REVIEW

**Comment by afishbeck** [comment]
Makes checking status on all roxie nodes optional from both command line and EclWatch.

**Comment by richardkchapman** [comment]
At a quick reading, this seems like it would change the behaviour for anyone using the old interface/not supplying a value for the new parameter to the SOAP service?

**Comment by afishbeck** [comment]
That's correct.  I can fix the default behaviors.  Only really helps if the default for EclWatch is to check only one roxie node.

Or, I could try making the EclWatch default check-all-nodes, but use  a cookie to remember if they've changed it and from then on for that user it would be faster.  People would have to be educated in order to benefit.

**Comment by richardkchapman** [comment]
It's ok for eclwatch and our other clients to be changed to default to checking one node, but you should not change the default behaviour of the SOAP interface (was the point I was trying to make)

**Comment by afishbeck** [comment]
@richardkchapman Updated.

File: esp/src/eclwatch/templates/QuerySetQueryWidget.html [hunk 1]
```diff
@@ -33,6 +33,22 @@
                             <option value="0">${i18n.NotActive}</option>
                         </select>
                     </div>
+                    <div id="${id}Options" data-dojo-type="dijit.form.DropDownButton">
+                        <span>${i18n.Options}</span>
+                        <div data-dojo-type="dijit.TooltipDialog">
+                            <div id="${id}OptionsForm" style="width: 530px;" onsubmit="return false;" data-dojo-type="dijit.form.Form">
+                                <div data-dojo-type="hpcc.TableContainer">
+			                        <select id="${id}CheckAllNodes" title="${i18n.Status}:" name="CheckAllNodes" colspan="2" data-dojo-type="dijit.form.Select">
```
Comment (miguelvazq) [inline esp/src/eclwatch/templates/QuerySetQueryWidget.html]: Minor comment on indentation here.

**Comment by miguelvazq** [comment]
Otherwise, looks good.

File: esp/src/eclwatch/templates/QuerySetQueryWidget.html [hunk 2]
```diff
@@ -33,6 +33,22 @@
                             <option value="0">${i18n.NotActive}</option>
                         </select>
                     </div>
```
Comment (miguelvazq) [inline esp/src/eclwatch/templates/QuerySetQueryWidget.html]: I would add <span data-dojo-type="dijit.ToolbarSeparator"></span> here to separate the controls on the toolbar.

**Comment by afishbeck** [comment]
@miguelvazq updated

**Comment by miguelvazq** [comment]
Looks good.

**Pull request merged**